submissions/kt-std/html-seat-booking/index.html

```diff
@@ -0,0 +1,505 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cinema booking</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Barcode+39+Extended+Text&family=Megrim&family=Quicksand:wght@300;500&display=swap" rel="stylesheet"> 
+  </head>
+
+  <body>
+    <main class="movie__container">
+
+      <!-- poster -->
+      <article class="poster__container ">
+        <button class="movie__play-button flex-center" name="button" value="Play trailer"><span class="play"></span></button>
+        <h2 class="poster__heading">ALIEN</h2>
```

Comment:
Avoid implementing visual representing in markup. Use CSS for it.

Reply:
Firstly I wanted to implement the `article` containing heading, image, and button to make something similar to the poster widget, but then I gave up the idea. Therefore, the heading is not needed anymore. Thank you for your advice! :)